src/main/java/AddDeadlineCommand.java

```diff
@@ -0,0 +1,23 @@
+import java.io.IOException;
+
+public class AddDeadlineCommand extends Command {
+    private String taskDescription;
+    private String date;
+
+    public AddDeadlineCommand(String description, String date) {
+        this.taskDescription = description;
```

Comment:
According to the [Java coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html)," `this` keyword should only be used when a field is shadowed by a method or constructor parameter". So, a better way would be either rename the constructor parameter to taskDescription and do this.taskDescription = taskDescription. Alternatively, you can do taskDescription = description :)